go.mod

```diff
@@ -22,21 +22,27 @@ require (
 	go.uber.org/zap v1.21.0
 	golang.org/x/sync v0.0.0-20220923202941-7f9b1623fab7
 	google.golang.org/grpc v1.49.0
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/client-go v0.25.2
+	k8s.io/api v0.25.4
+	k8s.io/apimachinery v0.25.4
+	k8s.io/client-go v0.25.4
 	knative.dev/eventing v0.35.2
 	knative.dev/eventing-kafka v0.35.1
 	knative.dev/eventing-kafka-broker v0.35.5
 	knative.dev/hack v0.0.0-20221010154335-3fdc50b9c24a
 	knative.dev/networking v0.0.0-20230224212030-2382b698cb6f
-	knative.dev/operator v0.34.3-0.20230309061722-d32ccab075c6
+	knative.dev/operator v0.35.6
 	knative.dev/pkg v0.0.0-20230224191730-67fb5f6e6735
 	knative.dev/serving v0.34.1
```

Comment:
this still points to 0.34?

Reply:
Stavros is working on a PR to bump operator.

Reply:
That's wrong and will be fixed.